Makefile

```diff
@@ -40,6 +40,7 @@ serve :
 test : build
 	$(DUNE) runtest $(DUNE_ARGS) src/parser/test
 	$(DUNE) build $(DUNE_ARGS) @test/model/runtest --no-buffer -j 1
+	$(DUNE) build $(DUNE_ARGS) @runtest --auto-promote --no-buffer -j 1
```

Comment:
this should be `$(DUNE) build $(DUNE_ARGS) @test/generators/runtest --no-buffer -j 1`